src/main/java/core/basesyntax/service/ShopServiceImpl.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.strategy.OperationStrategy;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class ShopServiceImpl implements ShopService {
+    private OperationStrategy operationStrategy;
+
+    public ShopServiceImpl(OperationStrategy operationStrategy) {
+        this.operationStrategy = operationStrategy;
+    }
+
+    @Override
+    public Map<String, Integer> process(List<FruitTransaction> transactions) {
+        Map<String, Integer> storage = new HashMap<>();
```

Comment:
Storage imitates DB so it is better to make it a public static map of the Storage class (optional)

Reply:
not fixed